lms/djangoapps/learner_dashboard/README.rst

```diff
@@ -1,12 +1,21 @@
-Learner Dashboard
-=================
+Status: Maintenance
+
+Responsibilities
+================
 
 This Django app hosts dashboard pages used by edX learners. The intent is for
 this Django app to include the following dashboard tabs:
 
  - Courses
  - Programs
 
+Direction: Deprecate
```

Comment:
Need to update the text below that says we should replace this with a "Backbone app".

Reply:
Done.